domain/domain_assets.bzl

```diff
@@ -0,0 +1,213 @@
+"""
+Macros for preparing & creating assets to include in the domain module.
+"""
+
+load("@rules_proto//proto:defs.bzl", "ProtoInfo")
+
+def _extract_proto_sources(deps):
+    """
+    Returns the list of proto source files that make up the specified list of proto dependencies.
```

Comment:
we can use the `#` for comments, nit fix, not blocking

Reply:
@anandwana001 I don't think we should be. See https://skydoc.bazel.build/docs/writing.html for examples on writing Skydocs. They use multi-line strings, not ``#``. I think we should be treating ``#`` as line comments and ``"""`` as documentation (which also matches Oppia web).

I did notice I'm not formatting on the correct line, but we can fix that in a later PR by making all documentation consistent.